low-level/sensors/impl/centos_7/systemd_watchdog.py

```diff
@@ -56,6 +56,8 @@
 from dbus.mainloop.glib import DBusGMainLoop
 import socket
 
+store = file_store

```

Comment:
reason ? node disks data status storage seems logical to reside on same server and not needed on node replacement. however any functional issue you saw without this ?

Reply:
If node is replaced, consul will still have old drive's data. That's why it is changed to file store

Reply:
ok, so same reason